src/Cleanup.php

```diff
@@ -312,6 +320,9 @@ public function postMigration(): void
 
     private function getDataToRemoveForRole(Connection $connection, string $name, string $mainRoleName): array
     {
+        if ($name === $mainRoleName) {
```

Comment:
tohle by nemělo nastat ale pro jistotu - kdyby to nastalo tak se smažou všechny projektový role a uživatele... to by bylo špatný